lib/live_debugger/live_views/sidebar_live.ex

```diff
@@ -104,12 +114,55 @@ defmodule LiveDebugger.LiveViews.SidebarLive do
           node_id={@node_id}
           highlight?={@highlight?}
           parent_lv_process={@parent_lv_process}
+          memory_raw_kb={@memory_raw_kb}
+          memory_gc_kb={@memory_gc_kb}
+          assigns_memory_kb={@assigns_memory_kb}
         />
       </.sidebar_slide_over>
     </div>
     """
   end
 
+  @impl true
+  def handle_info({:assign_memory_usage, pid, node_id}, socket)
+      when not is_nil(pid) and not is_nil(node_id) do
+    Process.send_after(self(), {:assign_memory_usage, pid, node_id}, @memory_poll_interval_ms)
+
+    raw =
+      case :erlang.process_info(pid, :memory) do
```

Comment:
Could you please create wrappers for these erlang function in `services/system` - we may want to mock them in the tests and there are some rumors that LiveView might get its own Debug module, so we want to have an easy way to swap them

Reply:
@kraleppa added in the wrappers. 

I did opt to skip the following in the ErlangService, but it can be easily added if you prefer.
```elixir
words = :erts_debug.size(assigns)
words * :erlang.system_info(:wordsize)
```